plugins/amazonq/codewhisperer/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codewhisperer/codescan/CodeWhispererCodeScanException.kt

```diff
@@ -5,33 +5,35 @@ package software.aws.toolkits.jetbrains.services.codewhisperer.codescan
 
 import software.aws.toolkits.resources.message
 
-open class CodeWhispererCodeScanException(override val message: String?) : RuntimeException()
+open class CodeScanException(override val message: String, open val code: String?) : RuntimeException(message)
```

Comment:
Should there be an enum for the code values possible?

Reply:
There can be, Let me raise a follow up CR addressing this along with more add on modifications from metrics.